language/analysis/README.md

```diff
@@ -30,12 +30,13 @@ mvn clean compile assembly:single
 ```
 
 We can then run the assembled JAR file with the `java` command. The variable $COMMAND takes
-three values `entities`, `sentiment` or `syntax`.
+six values `entities-text`, `entities-file`, `sentiment-text`, `sentiment-file`,
+`syntax-text`, or `syntax-file`.
```

Comment:
I think he just means in the command parser, I am in favor of simplifying the args parser for a lighter-weight usage string.